types/k6/encoding.d.ts

```diff
@@ -41,8 +57,23 @@ declare namespace encoding {
      * @example
      * encoding.b64decode(str)
      * encoding.b64decode(str, 'rawstd')
+     * const decBuffer = encoding.b64decode(str, 'rawurl')
+     * let decBin = new Uint8Array(decBuffer);
+     */
+    function b64decode(input: string, encoding?: Base64Variant): ArrayBuffer;
+    /**
+     * Base64 decode a string.
+     * https://k6.io/docs/javascript-api/k6-encoding/b64decode-input-encoding/
+     * @param input - Base64 encoded string or ArrayBuffer object.
+     * @param encoding - Base64 variant.
+     * @returns Decoded string.
+     * @example
+     * encoding.b64decode(str)
+     * encoding.b64decode(str, 'rawstd')
+     * const decBuffer = encoding.b64decode(str, 'rawurl')
+     * let decBin = new Uint8Array(decBuffer);
```

Comment:
Same here

Reply:
should be fixed by https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53101/commits/daf9b4e88aede66d57bc0721af49ec5c95616c89